### PR TITLE
Aea 6 payment info flow fixed victor

### PR DESCRIPTION
**Pull request by sdwvit**

### CODE REVIEW

File: lib/triggers/queryOrders.js [hunk 1]
```diff
@@ -148,6 +148,47 @@ function processTrigger(msg, cfg, next, snapshot) {
...
+
+        if (_.isEmpty(paymentIds)) {
+            return response;
+        }
+
+        function getPaymentIDs(orders) {
+            var paymentIds = _.compact(_.pluck(orders, 'paymentInfo'));
+
+            return _.uniq(paymentIds.map(function (element) {
+                return element.payments[0].id;
```
Comment (drobiazko) [inline lib/triggers/queryOrders.js]: What it there is a payment info with 0 payments?

File: lib/triggers/queryOrders.js [hunk 2]
```diff
@@ -148,6 +148,47 @@ function processTrigger(msg, cfg, next, snapshot) {
...
+            }));
+        }
+
+        function buildPaymentsHash(paymentsResponse){
+            return _.indexBy(paymentsResponse.body.results, 'id');
+        }
+
+        function addPaymentsToOrders(paymentsHash) {
+            _.each(response.body.results, function assignOrderPayment(order) {
+                order.payment = paymentsHash[order.paymentInfo.payments[0].id];
```
Comment (drobiazko) [inline lib/triggers/queryOrders.js]: Same here. A payment info with 0 payments will lead to an exception

File: lib/triggers/queryOrders.js (hunk 1 above)
Comment (sdwvit) [inline lib/triggers/queryOrders.js]: Fixed

File: lib/triggers/queryOrders.js (hunk 2 above)
Comment (sdwvit) [inline lib/triggers/queryOrders.js]: fixed

**Pull request merged**